website/client/components/groups/group.vue

```diff
@@ -636,7 +637,7 @@ export default {
     },
     async sendMessage () {
       let response = await this.$store.dispatch('chat:postChat', {
-        groupId: this.group._id,
+        group: this.group._id,
```

Comment:
Any reason you changed this? This should be groupId: https://github.com/HabitRPG/habitica/blob/develop/website/server/controllers/api-v3/chat.js#L119

Reply:
passing group is correct because I need the entire object here https://github.com/HabitRPG/habitica/pull/9023/files#diff-06de05f7a98f54b73bbc40c54accbf17R11 but I forgot to change `this.group._id` to `this.group`, will fix it now, the server is still getting sent groupId

Reply:
groupId in that controller is just the param name, it doesn't matter the way it's sent. Fixed the typo here https://github.com/HabitRPG/habitica/commit/d2832b71696f25cd3fbe63e2b9c676a764a70e07, thanks!